README.md

```diff
@@ -1 +1,166 @@
+
+- [📖 Overview](#about-project)
+  - [Project Requirements](#project-requirements)
+    - [API](#api)
+    - [Interfaces](#user-stories) 
+- [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [📸 Screenshots](#screenshots)
+  - [Desktop](#desktop)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## 📖 Overview <a name="about-project"></a>
+
+The LeaguesInfo WebApp is designed to give users a comprehensive look at their favorite leagues and teams. With our user-friendly interface, users can easily select their desired league and season to view a wealth of data on the teams, including stats such as goals and losses, as well as team names and logos. To ensure that we provide the most up-to-date and accurate information possible, we use the Football Standings API to fetch all of our data from the web. So whether you're a die-hard fan or just looking to stay informed, LeaguesInfo has everything you need to stay on top of the latest standings and stats for your favorite teams.
+
+### Project Objectives <a name="project-requirements"></a>
+
+#### API <a name="api"></a>
+
+For this project, I  use: 
++ Football Standings API: https://github.com/azharimm/football-standings-api
+
+
+#### Interfaces <a name="user-stories"></a>
+
+- The following interfaces were built:
+  - The *Leagues* page
+  - The *Standings* page
+- the given [layout](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) was followed with additional personalization(colors, typographies, spacings, etc).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+- [JavaScript/ES6](https://262.ecma-international.org/6.0/)
+- [npm](https://www.npmjs.com/)
+- [Git](https://git-scm.com/)
+- [GitHub](https://github.com)
+- [Stylelinter](https://stylelint.io/)
+- [ESLint](https://eslint.org/)
+- [SASS](https://sass-lang.com/)
+- [AXIOS](https://axios-http.com/docs/intro)
+- [React](https://react.dev/)
+- [Redux](https://redux.js.org/)
+- [Jest](https://jestjs.io/)
+
+### Key Features <a name="key-features"></a>
+
+The LeagueInfo application allows users to:
+
+- View a list of available leagues
+- Display a list of available seasons
+- View a list of available standings (teams and stats) of the current league and season
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+### Live Demo Link <a name="live-demo-link"></a>
+> [Live Demo Link](https://leaguesinfo.onrender.com/) <br>
+> [Video Demo](https://www.loom.com/share/8aaea0197571498494829a05e0f65148)
+
```

Comment:
- [ ] I will implore you to say what the purpose of this capstone is. For example `This is the React capstone project with the purpose of testing my knowledge on this this this...`. That was just an example but some thing related. Kindly make a new record to achieve this.